conf/conf.go

```diff
@@ -302,3 +311,34 @@ func SmtpCheck() {
 		panic(err.Error())
 	}
 }
+
+// GetLogLocation loads log file locations from a yaml file when called the first time.
+// Returns a struct with the log file locations.
+func GetLogLocation() *LogLocations {
+	logLocLock.Lock()
+	defer logLocLock.Unlock()
+
+	if logLoc == nil {
+		fc, err := ioutil.ReadFile(path.Join(resourcesPath, serverConfigFile))
```

Comment:
No problem since we are only running on unix, but technically it should be `filepath`.